### PR TITLE
LPS-187447 | Poshi Automation for Bug LPS-143405 | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Poshi Automation for Bug LPS-143405

Hi! :)
This pr is corresponding to this ticket: [LPS-143405](https://issues.liferay.com/browse/LPS-143405) → [LPS-187447](https://issues.liferay.com/browse/LPS-187447)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase [hunk 1]
```diff
@@ -3443,6 +3443,38 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-143405. Does return to Create Experience modal after cancel segment creation via experiences"
+	@priority = 3
+	test CancelSegmentCreationViaExperiencesModal {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase:3448]: CancelSegmentCreationDuringNewExperienceCreation

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase [hunk 2]
```diff
@@ -3443,6 +3443,38 @@ definition {
...
+		task ("Given the experience modal") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase:3449]: Given a page designer has a content page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase [hunk 3]
```diff
@@ -3443,6 +3443,38 @@ definition {
...
+			Click(
+				key_text = "New Experience",
+				locator1 = "Button#ANY");
+
+			Type(
+				locator1 = "PageEditor#EXPERIENCE_NAME",
+				value1 = "Experience Name");
+		}
+
+		task ("When we click to create a new segment and click the cancel button") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase:3465]: When the page designer cancel the segment creation during experience creation

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase [hunk 4]
```diff
@@ -3443,6 +3443,38 @@ definition {
...
+			Click(locator1 = "PageEditor#EXPERIENCE_CREATE_SEGMENT");
+		}
+
+		task ("Then it must return to the experience creation modal") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase:3469]: When the page designer should return to the experience creation modal

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase [hunk 5]
```diff
@@ -3443,6 +3443,38 @@ definition {
...
+			Button.clickCancel();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase:3470]: Move to when task

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase [hunk 6]
```diff
@@ -3443,6 +3443,38 @@ definition {
...
+
+			PageEditor.openExperienceMenu();
+
+			Click(
+				key_text = "New Experience",
+				locator1 = "Button#ANY");
+
+			Type(
+				locator1 = "PageEditor#EXPERIENCE_NAME",
+				value1 = "Experience Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase:3462]: Move to when task

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind refactoring according to comments? Thanks

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**